interfaces/mount/spec.go

```diff
@@ -39,10 +46,103 @@ func (spec *Specification) AddMountEntry(e Entry) error {
 	return nil
 }
 
+// resolveSpecialVariable resolves $SNAP, $SNAP_DATA and $SNAP_COMMON.
+func resolveSpecialVariable(path string, snapInfo *snap.Info) string {
```

Comment:
Nice! I feel though that the name of the function doesn't truly reflect its semantics, what it does is it interpolates special variables in the path string. As is, the name suggests it merely takes a name of the variable and expands it into a value.